python/requirements-wheel-test.txt

```diff
@@ -1,10 +1,11 @@
 cffi
 cython
 hypothesis
-numpy==1.19.4
-pandas<1.1.0; python_version < "3.8"
-pandas; python_version >= "3.8"
-pickle5; (python_version == "3.6" or python_version == "3.7") and sys_platform != 'win32'
+numpy==1.19.4; python_version != "3.9" or platform_machine != "aarch64"
+numpy==1.20.1; python_version == "3.9" and platform_machine == "aarch64"
```

Comment:
@xhochy despite that the build stage uses the same numpy versions, I'm getting `the` binary incompatibility error: `numpy.ndarray size changed, may indicate binary incompatibility. Expected 88 from C header, got 80 from PyObject`, but just on py39.
You may know the root cause of it, see the build log at https://travis-ci.com/github/ursacomputing/crossbow/builds/219762923#L3832

Reply:
cc @pitrou

Reply:
It may be a Pandas build problem @jorisvandenbossche

Reply:
Yeah, pandas is building from source on aarch64, since we don't have a wheel for the latest version. And numpy isn't pinned in our pyproject.toml for python 3.9, I see now (which should get fixed)

Reply:
Although numpy 1.20 should be the latest numpy anyway, at the moment. So that shouldn't matter?